content/news/036/index.md

```diff
@@ -482,7 +549,7 @@ After over half a year in development, Gamercade and related tools are ready
 for pre-alpha testing. This includes the [console](Gamercade-Console) itself,
```

Comment:
Here as well as the other paragraphs further down... Are they supposed to be formatted as [console][Gamercade-Console] ?